frontend/src/pages/api/fees-of-previous-7-days-utc.ts

```diff
@@ -11,6 +11,9 @@ const handler: NextApiHandler = async (req, res) => {
       const feeCountPerDay = await wampApi.getCall(
         nearNetwork
       )("nearcore-total-fee-count", [i]);
+      if (!feeCountPerDay) {
```

Comment:
@frol 
Is this a proper way?

Reply:
Can it actually return null-like value?

Reply:
No, but let's keep it this way in case anything goes wrong.